service/siteService.test.js

```diff
@@ -1,16 +1,117 @@
-const Airtable = require('airtable')
-
+const airtableClientMock = require('../airtableClient')
+const cacheServiceMock = require('./cacheService')
 const siteService = require('./siteService')
 
-describe("siteService tests", () => {
-    test('transforming list of public transit option strings returns list of publicTransitOption objects', () => {
+jest.mock('../airtableClient')
+jest.mock('./cacheService')
+
+describe("getMutualAidSites", () => {
+    afterEach(() => jest.resetAllMocks());
+
+    it('should return all properties accurately mapped', async () => {
+        const airtableResponse = [
+            {
```

Comment:
There's probably a cleaner way to do this or extract the json out into a file? Just left it in for now

Reply:
Yep, that's what I usually do in my apis, along with a combination of helper objects to easily create objects with only a couple different fields by just passing only those fields in, or libraries like podam that will just generate dummy test objects for you and then using an ojectmapper to turn them into json or something. I really dislike having json snippets or large object declarations directly in tests.

Just don't know how to do any of that in js haha, and also this is my very first experience with jest so trying to bite off manageable chunks at a time.

Reply:
Oooo, `podam` looks exciting.

To be honest, because JS objects are so... not Java objects, I think most of the work that would be required / bypassed in Java doesn't show up (Like class hierarchy and generics).

If you save the JSON as a file, my naive idea is that you could just read it in and just JSON it directly.  After that, the pattern I usually see is just nested dict/object comparison, which can be handled with libraries like `chai`.

However, no idea if that's still the current hotness.  You'd need to check with someone who actually knows JS (like @mc-funk!) for that 👍

Reply:
Yeah, that makes sense and is easy to do, it just gets messy and tedious if you're doing things like very specific field comparisons to have a bunch of json files laying around with very minor differences. buuuuuut a discussion that I'm sure will crop back up and can be tackled a different day :)